### ISSUE TITLE
About neeto libraries' ownership in npm

### ISSUE THREAD

**Issue by subins2000** [bigbinary/neeto-ui#869]
I was just checking npm package of neeto-ui and realized the ownership of the package is assigned to edwin’s personal account(?) . There are other bb related packages in the account too: https://www.npmjs.com/~edwinbabu

![image](https://user-images.githubusercontent.com/2305675/154942873-905a8ccc-e350-4154-ae2a-26c92473854c.png)

cc @yedhink

**Comment by yedhink** [comment]
@neerajdotname Can we use https://github.com/orgs/bigbinary/people/gitemit to publish these libraries and gems? This way even if someone leaves, the ownership of that package won't be attributed to someone who is not part of BigBinary.

If we can use gitemit, then please lemme know what's the email ID attached with `gitemit` Github account. This way I can use that email itself while signing up in npmjs. Thanks.

**Comment by neerajdotname** [comment]
@yedhink Sure let's do that.

@yedhink I'm sharing with you gitemit login info to github using a private channel.

**Comment by neerajdotname** [comment]
Let's reachout to Edwin and move over the ownership to either to gitemit or to `bot@bigbinary.com`.

**Comment by yedhink** [comment]
@amaldinesh7 @karthiknmenon 
I have added `bot@bigbinary.com`, with username `bigbinarybot`, as the owner and publisher in npmjs. So from now on, that account should be the one that takes care of publishing the npm registry.

I tested our release action via #874.

<img width="1193" alt="image" src="https://user-images.githubusercontent.com/31244999/155118253-62818d92-005f-4b73-a1f6-3abf2e873172.png">

@sangameshsomawar I will be adding the same logistics in https://github.com/bigbinary/neeto-ui-rn too.
